smoketest/__init__.py

```diff
@@ -129,7 +132,14 @@ def main():
 
     directives = []
     for filename in args.input_filenames:
-        directives.extend(generate_directives_from_file(filename, args))
+        try:
+            directives.extend(generate_directives_from_file(filename, args))
+        except InputFileError as e:
+            print('Smoketest had a problem with the input file {0}:'.format(
```

Comment:
Nitpick: it may be a little clearer to put the file name in quotes.

Reply:
Done in 1d58b504.